src/libs/actions/Session/index.js

```diff
@@ -642,6 +621,23 @@ function authenticatePusher(socketID, channelName, callback) {
     });
 }
 
+function getShortLivedAuthToken() {
+    // TODO: If we have any way to read the shortLivedAuthToken and save it, we can also no longer call makeRequestWithSideEffects.
+    // eslint-disable-next-line rulesdir/no-api-side-effects-method
+    API.makeRequestWithSideEffects(
+        'OpenOldDotLink', {shouldRetry: false}, {},
+    ).then((response) => {
+        const {shortLivedAuthToken} = response;
+        Onyx.merge(ONYXKEYS.SESSION, {shortLivedAuthToken});
```

Comment:
**original context:** https://github.com/Expensify/Expensify/issues/218742

Reply:
Yeah it looks like this should just be an `API.read` now. We're just listening for the short lived token to change in Onyx instead of blocking on the request.